js/src/widgets/metadataView.js

```diff
@@ -208,30 +219,77 @@
   },
 
     bindEvents: function() {
```

Comment:
Subscribe to detect changes in the annotations list, IE: the focused canvas may have changed.